katas/percy/langtonant/README.md

```diff
@@ -0,0 +1,16 @@
+# Langton's Ant!
```

Comment:
hello Percy, solution files location needs to be fixed, in this case, please follow instructions in `katas/README` file, so it follows the convention `katas/<kata>/<your name>/` for the container folder. Also, no need to duplicate README file for each kata, just add .js solution files